crates/cdk/src/mint/melt.rs

```diff
@@ -439,40 +407,26 @@ impl Mint {
             }
         }
 
-        let quote = match self.verify_melt_request(melt_request).await {
-            Ok(quote) => quote,
-            Err(err) => {
-                tracing::debug!("Error attempting to verify melt quote: {}", err);
+        let mut tx = self.localstore.begin_transaction().await?;
```

Comment:
Since we start the transaction here and we do not commit until after the payment has been completed the database will be locked for the entire time it takes to make the ln payment. Most ln payments are fast but not all in the event one gets stuck the whole mint db will be blocked waiting on the one tx, I think we need to release the db lock while we wait for the ln payment.